_episodes/02-loop.md

```diff
@@ -149,10 +157,10 @@ where each character (`char`) in the variable `word` is looped through and print
 The numbers in the diagram denote which loop cycle the character was printed in (1 being the first loop, and 6 being the final loop).
 
 We can call the [loop variable]({{ page.root }}/reference/#loop-variable) anything we like,
-but there must be a colon at the end of the line starting the loop,
-and we must indent anything we want to run inside the loop. Unlike many other languages, there is no
-command to signify the end of the loop body (e.g. `end for`); what is indented after the `for` statement belongs to the loop.
-
+but to open the block we can't forget to put a colon at the end of the line starting the loop,
```

Comment:
same as above: `block` is jargon. 
This section's initial content was fine as is. I agree that some context on the purpose colon would be helpful but not by increasing the cognitive load of the section.